Core/include/Acts/Propagator/StandardAborters.hpp

```diff
@@ -101,14 +101,11 @@ struct SurfaceReached {
       return true;
     }
 
-    // not blindly using the stepper overstep limit here because it does not
-    // always work for perigee surfaces.
+    // not using the stepper overstep limit here because it does not always work
+    // for perigee surfaces
     // note: the near limit is necessary for surfaces with more than one
-    // intersections in order to discard the ones which are behind us.
-    const double nearLimit =
-        overrideNearLimit.value_or(stepper.overstepLimit(state.stepping));
-    const double farLimit =
-        state.stepping.stepSize.value(ConstrainedStep::aborter);
+    // intersection in order to discard the ones which are behind us
```

Comment:
But the near limit is removed, is the comment still valid?

Reply:
For the `SurfaceReached` we still use the `nearLimit`. I wanted to drop it but when we target a cylinder with the `SurfaceReached` aborter it would break due to the ambiguity of the intersections.